compiler/src/dmd/cli.d

```diff
@@ -459,7 +459,7 @@ dmd -cov -unittest myprog.d
              $(P Note that multiple `-i=...` options are allowed, each one adds a pattern.)}"
         ),
         Option("ignore",
-            "ignore unsupported pragmas"
+            "deprecated flag, unsupported pragmas are always ignored now"
```

Comment:
Ignored even with `-w`?

Reply:
Yes

Reply:
Ok, I was thinking of the duality in gdc _because_ of the frontend accepting unknown pragmas to gdc.

https://github.com/gcc-mirror/gcc/blob/ff8d0ce17fb585a29a83349acbc67b2dd3556629/gcc/d/decl.cc#L374